docs/src/api/class-fetchrequest.md

```diff
@@ -108,9 +111,12 @@ Query parameters to be send with the URL.
 Allows to set HTTP headers.
 
 ### option: FetchRequest.post.data
-- `data` <[string]|[Buffer]>
+- `data` <[string]|[Buffer]|[Serializable]>
```

Comment:
formData is a bit more complicated than that:
https://developer.mozilla.org/en-US/docs/Web/API/FormData/append